searchinrotatedsortedarray/search-in-rotated-sorted-array.js

```diff
@@ -0,0 +1,19 @@
+// https://leetcode.com/problems/search-in-rotated-sorted-array/
+function searchInRotatedSortedArray(nums,target){
+    const array = nums.concat(target)
+    const mySet = new Set()
+    for(let i=0; i<array.length; i++){
```

Comment:
zamien to na lepszego fora, starajmy sie odchodzic od prymitywnego fora tak czesto jak to mozliwe